addons/mail/static/src/views/web/fields/html_mail_field/html_mail_field.js

```diff
@@ -17,7 +17,14 @@ export class HtmlMailField extends HtmlField {
         }
         const cssRules = cssRulesByElement.get(editor.editable);
         el.classList.remove("odoo-editor-editable");
+        const iframe = document.createElement('iframe');
+        iframe.style.height = '0px';
+        iframe.style.visibility = 'hidden';
+        iframe.setAttribute('sandbox', 'allow-same-origin'); // Make sure no scripts get executed.
+        iframe.appendChild(el);
+        document.body.append(iframe);
```

Comment:
Would add a comment to say exactly why we need the iframe. e.g. "//Put element inside an iframe so we can get its computed style"

Reply:
> Would add a comment to say exactly why we need the iframe. e.g. "//Put element inside an iframe so we can get its computed style"

The comment has been added as per your suggestions. Thank you :)

Reply:
@mega-odoo now begins your quest to find someone with r+ :vulcan_salute: